### PR TITLE
Fix is touch check for android

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard22**

### CODE REVIEW

File: packages/app/src/utils/use-is-touch-device.tsx [hunk 1]
```diff
@@ -13,7 +13,11 @@ export function IsTouchDeviceContextProvider({
 }: {
   children: ReactNode;
 }) {
-  const isTouchDevice = useMediaQuery('(hover: none)');
+  let isTouchDevice = false;
+
+  if (typeof window !== 'undefined') {
+    isTouchDevice = isTouch();
+  }
```
Comment (VWSCoronaDashboard24) [inline packages/app/src/utils/use-is-touch-device.tsx:20]: Maybe move this to the hook? We can just return `false` when no `window` is available. Now there are two places with logic.

File: packages/app/src/utils/is-touch.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+/**
+ * Check wether a user is using a touch device.
+ * @returns boolean
+ */
+export function isTouch() {
```
Comment (VWSCoronaDashboard24) [inline packages/app/src/utils/is-touch.ts:5]: Maybe rename to `isTouchCapableDevice`? Or `isTouchDevice`?

Comment (VWSCoronaDashboard22) [reply packages/app/src/utils/is-touch.ts]: Renamed

File: packages/app/src/utils/use-is-touch-device.tsx (hunk 1 above)
Comment (VWSCoronaDashboard22) [reply packages/app/src/utils/use-is-touch-device.tsx]: Moved logic to isTouchDevice

**Review by VWSCoronaDashboard19 — APPROVED** [review]

**Review by VWSCoronaDashboard24 — APPROVED** [review]

**Pull request merged**